metriq_gym/benchmarks/bseq.py

```diff
@@ -26,7 +26,6 @@
 from metriq_gym.qplatform.device import connectivity_graph
 
 
-@dataclass
```

Comment:
Do we plan on doing a similar type of adaptation to the `*Result` classes for the other benchmarks?

Reply:
well spotted